calico-enterprise/reference/resources/bgpconfig.mdx

```diff
@@ -66,6 +66,7 @@ spec:
 | prefixAdvertisements   | List of per-prefix advertisement properties, like BGP communities.                                                                                                                                                                                                                                                                                                                                                             |                                                         | List of [prefixAdvertisements](#prefixadvertisements) |
 | nodeMeshPassword       | BGP password for the all the peerings in a full mesh configuration.                                                                                                                                                                                                                                                                                                                                                            |                                                         | [BGPPassword](bgppeer.mdx#bgppassword)                | `nil` (no password)                                             |
 | nodeMeshMaxRestartTime | Restart time that is announced by BIRD in the BGP graceful restart capability and that specifies how long the neighbor would wait for the BGP session to re-establish after a restart before deleting stale routes in full mesh configurations. Note: extra care should be taken when changing this configuration, as it may break networking in your cluster. When not specified, BIRD uses the default value of 120 seconds. | `10s`, `120s`, `2m` etc.                                | [Duration string][parse-duration]                     | `nil` (empty config, BIRD will use the default value of `120s`) |
+| ignoredInterfaces      | List of network interfaces to be excluded when reading device routes.                                                                                                                                                                                                                                                                                                                                                          | A list of network interface names. The names can contain the wildcard character asterisk `*` to specify groups of interface names. | List of string                                        | `nil` (no extra interfaces to be ignored)                       |
```

Comment:
List of string --> List of strings?

Reply:
Well, my initial thought was same as you. But then I saw this one and thought maybe there is a merit to "string"
https://github.com/tigera/docs/blob/ddc5f485f7269a28018c2e10c1da34a9a92bd8d7/calico/reference/resources/bgpconfig.mdx?plain=1#L66
I'm happy to change it to `strings` given that is the correct one in my opinion as well.